offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/affiliation/create/CreateAffiliationView.groovy

```diff
@@ -25,385 +23,106 @@ import java.util.stream.Collectors
-        addressAdditionField.setPlaceholder("Department, Faculty, or other specification of affiliation name")
-        addressAdditionField.setDescription("In case the affiliation differs from the organisation you can further specify that here.")
-
-        this.streetField = new TextField("Street")
-        streetField.setPlaceholder("Street name and street number ")
-
-        this.postalCodeField = new TextField("Postal Code")
-        postalCodeField.setPlaceholder("Customer postal code")
-
-        this.cityField = new TextField("City")
-        cityField.setPlaceholder("Name of the city")
-
-        this.countryBox = new ComboBox<>("Country")
-        countryBox.setPlaceholder("Name of the country")
-        countryBox.setDescription("Select the name of the country e.g. Germany")
-        countryBox.setItems(Country.availableCountryNames())
-
-        this.affiliationCategoryField = generateAffiliationCategorySelect(createAffiliationViewModel.affiliationCategories) as ComboBox<String>
 
         this.abortButton = new Button("Abort Affiliation Creation")
         abortButton.setIcon(VaadinIcons.CLOSE_CIRCLE)
         abortButton.addStyleName(ValoTheme.BUTTON_DANGER)
 
         this.submitButton = new Button("Create Affiliation")
-        submitButton.enabled = allValuesValid()
+        submitButton.enabled = false
         submitButton.setIcon(VaadinIcons.OFFICE)
         submitButton.addStyleName(ValoTheme.BUTTON_FRIENDLY)
 
-        organisationBox.setRequiredIndicatorVisible(true)
-        addressAdditionField.setRequiredIndicatorVisible(false)
-        streetField.setRequiredIndicatorVisible(true)
-        postalCodeField.setRequiredIndicatorVisible(true)
-        cityField.setRequiredIndicatorVisible(true)
-        countryBox.setRequiredIndicatorVisible(true)
-        affiliationCategoryField.setRequiredIndicatorVisible(true)
-
-        HorizontalLayout row1 = new HorizontalLayout(organisationBox, addressAdditionField)
-        row1.setSizeFull()
-        HorizontalLayout row2 = new HorizontalLayout(streetField)
-        row2.setSizeFull()
-        HorizontalLayout row3 = new HorizontalLayout(postalCodeField, cityField)
-        row3.setSizeFull()
-        row3.setExpandRatio(postalCodeField, 1)
-        row3.setExpandRatio(cityField, 3) // leads to it being 3/4 of the width
-        HorizontalLayout row4 = new HorizontalLayout(countryBox)
-        row4.setSizeFull()
-
         HorizontalLayout buttonLayout = new HorizontalLayout(abortButton, submitButton)
-        HorizontalLayout row5 = new HorizontalLayout(affiliationCategoryField, buttonLayout)
-        row5.setComponentAlignment(affiliationCategoryField, Alignment.BOTTOM_LEFT)
-        row5.setComponentAlignment(buttonLayout, Alignment.BOTTOM_RIGHT)
-        row5.setSizeFull()
+        buttonLayout.setComponentAlignment(abortButton, Alignment.MIDDLE_RIGHT)
+        buttonLayout.setComponentAlignment(submitButton, Alignment.MIDDLE_RIGHT)
```

Comment:
The layout for the abort button should be as it was before on the bottom right next to the create button